### ISSUE TITLE
Add URL tests for SVG and MathML elements

### ISSUE THREAD

**Issue by sideshowbarker** [w3c/web-platform-tests#1186]
See https://critic.hoppipolla.co.uk/showcomment?chain=4284

**Comment by annevk** [comment]
@gsnedders it seems this should not have been closed. Those commits are for a different repository.

**Comment by GPHemsley** [comment]
The link here is dead. What are the requirements here? Is this still necessary?

**Comment by sideshowbarker** [comment]
> @sideshowbarker The link here is dead. What are the requirements here? Is this still necessary?

Yes, still necessary as far as I know. At least I know of nothing changed that would have made it any less necessary than it was when the issue was created as a result of review for the URL tests.

I guess the requirements are to have a driver that takes the data from https://github.com/web-platform-tests/wpt/blob/master/url/resources/urltestdata.json and uses it to test parsing of URLs in SVG and MathML content (whereas the current tests are just testing that for HTML).

**Comment by annevk** [comment]
@sideshowbarker so this isn't so much about conformance checkers as the label indicates?

**Comment by sideshowbarker** [comment]
> @sideshowbarker so this isn't so much about conformance checkers as the label indicates?

ah yeah I misspoke it is that instead. So I just now retitled it to make that clear
